package.json

```diff
@@ -2,19 +2,34 @@
   "name": "@erc725/erc725.js",
   "version": "0.21.3",
   "description": "Library to interact with ERC725 smart contracts",
-  "main": "build/main/src/index.js",
-  "typings": "build/main/src/index.d.ts",
-  "module": "build/module/src/index.js",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
```

Comment:
What is the benefit of having a `dist` folder instead of the old `build/`? Maybe we can configure the build system to keep `build/` folder still? But not a big deal

Reply:
dist is just a bit more standard and doesn't require any configuration of tsup, but you're right either way would work. I will make a patch to make sure this all works as expected.

Reply:
I am reverting the dist vs build folder. Initially it was causing problems within mocha and also tsup. Since it really a smaller library and doesn't quite need tsup and pnpm and dist. It will remove some dependencies and simplify this PR as well.